calitp/__init__.py

```diff
@@ -1,6 +1,6 @@
 # flake8: noqa
 
-__version__ = "0.0.12"
```

Comment:
I don't see a 13 or 14 release, I think 0.0.13 should be fine?

Reply:
I messed up the tags for those so they already exist associated with the wrong commits :(

Reply:
you can always delete tags and also I think tags matter less than releases

but idc that much, 15 is fine

Reply:
hmm yeah I was looking for how to delete the tags but couldn't find it...

Reply:
hah nevermind just found it, deleting tags and reducing the version number in this PR